Add new Cookies and Privacy Policy page in Markdown format

LeaWCO opened the pull request:
CD changes based on updated PA, PO and ID input. Includes GA4 clarification, Consent Mode, and user preference management information. Content rewritten to follow GDS style for clarity and plain English.

LeaWCO committed:
Add new Cookies and Privacy Policy page in Markdown format

CD changes based on updated PA, PO and ID input. Includes GA4 clarification, Consent Mode, and user preference management information.
Content rewritten to follow GDS style for clarity and plain English.

LeaWCO commented on src/_pages/Cookies and privacy policy.md:
```diff
@@ -0,0 +1,53 @@
+
+\#\# Managing your cookie preferences
+
+When you visit \[Service Name\], we will ask if you want to accept analytics cookies.
```
When you visit 'find support after a fit note', we will ask if you want to accept analytics cookies.

trevorsaint replied:
@LeaWCO can you update the markdown file to use correct markdown syntax. I'm not sure if those back slashes have been brought in from Google Docs as a copy and paste.

Also can you keep the file name the same as it was earlier. "cookies.md" Or even "cookies-and-privacy.md"

The page name is not determined by the file name but the title in the meta data frontmatter.

LyndsayOBrien2 replied:
@LeaWCO @trevorsaint - should I leave the review until after the changes Trevor has suggested are made?

LeaWCO reviewed (commented):
Hi @trevorsaint I forgot to add in the research pilot name

LeaWCO committed:
Corrected cookies and privacy policy markdown and filename

- Corrected Markdown formatting for the cookies and privacy policy page
- Replaced placeholder text ([Service Name]) with actual name
- Added missing link for managing cookie preferences
- Renamed the file to lowercase, hyphenated format: `cookies-and-privacy-policy.md`
- Saved the file inside the correct `src/_pages/` directory

To match the expected file standards and Markdown style used across the project, based on feedback from the interaction design team.

LeaWCO committed:
Corrected cookies and privacy policy markdown and filename

## What I did

- Corrected Markdown formatting for the cookies and privacy policy page
- Replaced placeholder text ([Service Name]) with actual name
- Added missing link for managing cookie preferences
- Renamed the file to lowercase, hyphenated format: `cookies-and-privacy-policy.md`
- Saved the file inside the correct `src/_pages/` directory

## Why

To match the expected file standards and Markdown style used across the project, based on feedback from the interaction design team.

trevorsaint reviewed (changes requested):
Hi @LeaWCO I see that we have a null link in the page.

We're on the cookies page, so we can change the wording.

Also what is the ":" in the table headers? This is not needed.

Thanks.

LeaWCO commented:
> Hi @LeaWCO I see that we have a null link in the page.
> 
> We're on the cookies page, so we can change the wording.
> 
> Also what is the ":" in the table headers? This is not needed.
> 
> Thanks.

Hey Trevor, happy for you to correct minor, tiny and uncontentious things - thank you

trevorsaint committed:
Fixed cookies page. Added back in the correct frontmatter code

trevorsaint reviewed (approved):
@LeaWCO looks good now.

trevorsaint commented:
@LyndsayOBrien2 you can review it now if you like.

LeaWCO reviewed (commented):
Thanks Trevor, all good

LyndsayOBrien2 reviewed (approved):
Looks good to me thanks @trevorsaint  and @LeaWCO
